code/game/objects/structures/displaycase.dm

```diff
@@ -138,7 +138,8 @@
 
 
 /obj/structure/displaycase/bullet_act(var/obj/item/projectile/Proj)
-	health -= Proj.damage
+	if(!istype(Proj ,/obj/item/projectile/beam/lasertag) && !istype(Proj ,/obj/item/projectile/beam/practice) )
```

Comment:
1. What the FUCK is up with your spaces?
2. Don't those projectiles deal 0 damage anyway? At least they should.
3. I'm 90% sure projectiles such as these two have a variable specifying they should never deal damage. Use that rather than a hardcoded type check.